app/protocol/flows/transactionrelay/handleRequestedTransactions_test.go

```diff
@@ -0,0 +1,88 @@
+package transactionrelay_test
+
+import (
+	"github.com/kaspanet/kaspad/app/appmessage"
+	"github.com/kaspanet/kaspad/app/protocol/flows/transactionrelay"
+	"github.com/kaspanet/kaspad/domain"
+	"github.com/kaspanet/kaspad/domain/consensus"
+	"github.com/kaspanet/kaspad/domain/consensus/model/externalapi"
+	"github.com/kaspanet/kaspad/domain/consensus/utils/testutils"
+	"github.com/kaspanet/kaspad/domain/dagconfig"
+	"github.com/kaspanet/kaspad/infrastructure/config"
+	"github.com/kaspanet/kaspad/infrastructure/logger"
+	"github.com/kaspanet/kaspad/infrastructure/network/netadapter"
+	"github.com/kaspanet/kaspad/infrastructure/network/netadapter/router"
+	"github.com/kaspanet/kaspad/util/panics"
+	"github.com/pkg/errors"
+	"testing"
+)
+
+// TestHandleRequestedTransactions verifies that the flow of  HandleRequestedTransactions
```

Comment:
This test doesn't check the main part of the flow - when the transaction is available.
Please include this in this test, or in a separate test.